mechanical/body_assembly/laser_cut_parts/README.md

```diff
@@ -0,0 +1,24 @@
+# Replacement Top and Bottom Plates
+
+## The Need for Replacement Plates
+The 12x9 aluminum pattern plates specified in other parts of the rover documentation have been discontinued by ServoCity. The included *top\_plate\_baseline.dxf* and *bottom\_plate\_baseline.dxf* can serve as replacements.
+
+These plates are the same size as the originals, so everything else is the same except for items noted below.
+
+## Cutting the Plates
```

Comment:
i really think we need default instructions here, not just suggestions of what can be done. People are going to be approaching this for the first time not knowing what they're doing

Reply:
I also added a note in section 3 of the body_assembly README pointing to the information in laser_cut_parts. I supposed we can add others.

Reply:
Well, I'm saying we should settle on one material as the default. You say either plywood or mdf below.

Reply:
It doesn't matter which you use. If the material is at least 5mm think and either plywood or MDF, then you are good. I love the idea of people being able to customize their rovers beyond just the color(s) of the 3D-printed plastic parts.

Reply:
If it doesn't matter, we should just put one as a default. Simplify for people going through this for the first time. Customization is all well and good, but put a default for people to start with.

Reply:
It's a bit more complicated than that. MDF would be a good default if you are using Sculpteo because it's slightly cheaper and you get lots of color choices. MDF is a harder call if you are cutting it yourself because it can have issues with formaldehyde if you are not somewhat careful about suppliers. (I know of someone who's laser caught on fire presumably because she was using dodgy MDF.) So you don't want someone who has plenty of perfectly acceptable plywood around going out and buying MDF if they don't need to. I'll add some extra comments that might strike a happy middle ground.

Reply:
Updated discussion looks fine, seems sufficient to me.

Reply:
In case anyone doesn't know, formaldehyde is a known carcinogen and really needs to be painted on all sides to stop the formaldehyde from leaching out!

Reply:
> In case anyone doesn't know, formaldehyde is a known carcinogen and mdf needs to be painted on all sides to stop the formaldehyde from leaching out!

This is a really good point. It's even worse when laser cutting MDF because the heat from the laser sets the formaldehyde free all at once, which can be toxic. Most people who cut MDF use boards that meet California requirements for formaldehyde, which are lower than the EPA requirements. You can get MDF that has little or no formaldehyde, but it takes some work to find it. This is one of the reasons I recommend people who want to use MDF to use a service rather than trying to cut their own. I will add a note that people should paint their MDF assuming it doesn't already come painted. I think most people will do that anyway, because bare MDF is sort of blah. Formaldehyde can be a problem with plywood too, but it's relatively easy to find project-grade plywood that has little or no formaldehyde.